### DISCUSSION TITLE
DENT VM image

### DISCUSSION THREAD

**Discussion by tsmjay** [dentproject/dentOS#273] [General]
Hi,

I notice in the meeting minutes from the weekly call, that there is mention of KVM images.

In the [documentation](https://github.com/opencomputeproject/onie/blob/master/machine/kvm_x86_64/INSTALL) for ONIE, it mentions running ONIE inside QEMU/KVM.

Does this mean that it's possible to run DENT inside a VM?

Thanks.

**Comment by taraschornyiplv** [comment]
Yes,
You can find information about how here
https://github.com/dentproject/dentOS/wiki/How-to-install-Dent-on-a-virtual-machine-manager
